2021-08-28 11:12:37,350 ERROR: Exception on /admin [GET] [in c:\users\fb\appdata\local\programs\python\python37\lib\site-packages\flask\app.py:1455]
Traceback (most recent call last):
  File "c:\users\fb\appdata\local\programs\python\python37\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\fb\appdata\local\programs\python\python37\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\fb\appdata\local\programs\python\python37\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\fb\appdata\local\programs\python\python37\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\xampp\htdocs\onely\app\main\routes.py", line 28, in admin
    return render_template('admin.html', all_users=all_users)
  File "c:\users\fb\appdata\local\programs\python\python37\lib\site-packages\flask\templating.py", line 150, in render_template
    ctx.app,
  File "c:\users\fb\appdata\local\programs\python\python37\lib\site-packages\flask\templating.py", line 128, in _render
    rv = template.render(context)
  File "c:\users\fb\appdata\local\programs\python\python37\lib\site-packages\jinja2\environment.py", line 1304, in render
    self.environment.handle_exception()
  File "c:\users\fb\appdata\local\programs\python\python37\lib\site-packages\jinja2\environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\xampp\htdocs\onely\app\templates\admin.html", line 36, in top-level template code
    {% for key, segment in all_users.items() %}
  File "c:\users\fb\appdata\local\programs\python\python37\lib\site-packages\jinja2\utils.py", line 84, in from_obj
    if hasattr(obj, "jinja_pass_arg"):
jinja2.exceptions.UndefinedError: 'list object' has no attribute 'items'
2021-08-28 11:12:56,010 INFO: Strefi startup [in C:\xampp\htdocs\onely\app\__init__.py:90]
2021-08-28 11:12:57,536 ERROR: Exception on /admin [GET] [in c:\users\fb\appdata\local\programs\python\python37\lib\site-packages\flask\app.py:1455]
Traceback (most recent call last):
  File "c:\users\fb\appdata\local\programs\python\python37\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\fb\appdata\local\programs\python\python37\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\fb\appdata\local\programs\python\python37\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\fb\appdata\local\programs\python\python37\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\xampp\htdocs\onely\app\main\routes.py", line 28, in admin
    return render_template('admin.html', all_users=all_users)
  File "c:\users\fb\appdata\local\programs\python\python37\lib\site-packages\flask\templating.py", line 150, in render_template
    ctx.app,
  File "c:\users\fb\appdata\local\programs\python\python37\lib\site-packages\flask\templating.py", line 128, in _render
    rv = template.render(context)
  File "c:\users\fb\appdata\local\programs\python\python37\lib\site-packages\jinja2\environment.py", line 1304, in render
    self.environment.handle_exception()
  File "c:\users\fb\appdata\local\programs\python\python37\lib\site-packages\jinja2\environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\xampp\htdocs\onely\app\templates\admin.html", line 36, in top-level template code
    {% for key, segment in all_users %}
ValueError: too many values to unpack (expected 2)
2021-08-28 11:14:03,588 INFO: Strefi startup [in C:\xampp\htdocs\onely\app\__init__.py:90]
2021-08-28 11:14:47,059 INFO: Strefi startup [in C:\xampp\htdocs\onely\app\__init__.py:90]
2021-08-28 11:15:53,920 INFO: Strefi startup [in C:\xampp\htdocs\onely\app\__init__.py:90]
2021-08-28 11:16:45,683 INFO: Strefi startup [in C:\xampp\htdocs\onely\app\__init__.py:90]
2021-08-28 11:19:30,042 INFO: Strefi startup [in C:\xampp\htdocs\onely\app\__init__.py:90]
2021-08-28 11:20:09,304 INFO: Strefi startup [in C:\xampp\htdocs\onely\app\__init__.py:90]
2021-08-28 11:21:12,999 INFO: Strefi startup [in C:\xampp\htdocs\onely\app\__init__.py:90]
2021-08-28 11:21:35,930 INFO: Strefi startup [in C:\xampp\htdocs\onely\app\__init__.py:90]
2021-08-28 11:22:05,762 INFO: Strefi startup [in C:\xampp\htdocs\onely\app\__init__.py:90]
